lib/gorgon/originator.rb

```diff
@@ -30,13 +30,14 @@ def originate
       publish
       @logger.log "Originator finished successfully"
     rescue StandardError
-      puts "Unhandled exception in originator:"
-      puts $!.message
-      puts $!.backtrace.join("\n")
-      puts "----------------------------------"
-      puts "Now attempting to cancel the job."
+      $stderr.puts "Unhandled exception in originator:"
+      $stderr.puts $!.message
+      $stderr.puts $!.backtrace.join("\n")
+      $stderr.puts "----------------------------------"
+      $stderr.puts "Now attempting to cancel the job."
       @logger.log_error "Unhandled Exception!" if @logger
       cancel_job
+      exit 2
```

Comment:
1 was used for when you input an invalid argument to `gorgon`.

Admittedly I don't think these would ever get used for anything meaningful (and if they are, we should probably do it by setting constants in a central file), but now scripts do have the ability to differentiate between the types of errors they're encountering.